src/examples/java/com/amazonaws/crypto/examples/SimpleDataKeyCachingExample.java

```diff
@@ -1,72 +0,0 @@
-// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
I think this example is largely handled by https://github.com/acioc/aws-encryption-sdk-java/blob/keyring/src/examples/java/com/amazonaws/crypto/examples/cryptomaterialsmanager/caching/SimpleCache.java already

Reply:
This is actually part of the reason I created https://github.com/aws/aws-encryption-sdk-java/issues/225

I noticed we have a number of duplicates between legacy and what is already there. And the wording between different examples (legacy v keyring vs ones I wrote) is inconsistent as well. So we should have a separate task to actually finalize what examples we need, what their wording should be, and removing duplicates.

Right now, duplicates are fine since they all run anyways.

Reply:
Calling out this specifically because it's in the docs. Agreed that we should coordinate on finalizing these with June.